src/main/default/lwc/lookup/lookup.js

```diff
@@ -14,9 +13,11 @@ export default class Lookup extends LightningElement {
     @api isMultiEntry = false;
     @api hideResults = false;
     get showResults() {return this.isMultiEntry && !this.hideResults}
-    @api errors = [];
     @api scrollAfterNItems;
     @api iconResources = {};
+    @api errors = [];
+    @api disabled;
+    @api minimalSearchTermLength = 0;
```

Comment:
the default should stay at 2

Reply:
how about 
@api minimalSearchTermLength = MINIMAL_SEARCH_TERM_LENGTH;
to not change so many things in this component, but make it more extending original code instead of replacing it,
this will be much easier to merge and maintain